### ISSUE TITLE
Set v-model for <v-select multiple></v-select> doesn't work

### ISSUE THREAD

**Issue by zhanghaowx** [johnleider/vue-materials#30]
Version: 1.0.3-rc.4

When I use `<v-select v-model="selectedItems" multiple></v-select>` and selectedItems is set an as array of values. The component doesn't respect initial values in selectedItems.

Also, I think it would be worth noting in the docs that the `:items` property should be like
```
[
    {
        id: "id1",
        text: "ID 1"
    }
]
```

This is a great library, thanks for the work!

**Comment by johnleider** [comment]
Could you provide an example of your markup, the expected behavior, and what resulting behavior please? Also, thank you for the documentation catch.

**Comment by zhanghaowx** [comment]
Here is an example:

```
<div class="input-field">
    <v-select name="jobFilter" id="jobFilter" :items="jobFilterOptions" v-model="statesFiltered" multiple></v-select>
    <label for="jobFilter">Display Jobs</label>
</div>
```

```
export default {
    data() {
        return {
            states: ["active", "wait", "completed", "failed"],
            statesFiltered: ["active", "wait", "completed", "failed"],
        }
    },
    computed: {
        jobFilterOptions: function() {
            return this.states.map((state, i) => {
                return {
                    id: state,
                    text: state,
                }
            })
        },
    }
}
```

Ideally the multiple selector should have all checkboxes checked initially because that's the initial value in statesFiltered. However none of them get checked

**Comment by johnleider** [comment]
ty! I'll get this fixed asap.

**Comment by johnleider** [comment]
This has been fixed with commit [43ea5e0](https://github.com/johnleider/vue-materials/commit/43ea5e0a4127dc8089a66275aec9db1bb8730e01).
